version: "3.8"

services:

  harvester_test:
    build:
      context: .
      dockerfile: Dockerfile
    restart: "on-failure"
    entrypoint: python3
    command: -m unittest discover
